examples/meteor/client/styles/base.css

```diff
@@ -285,9 +284,9 @@ label[for='toggle-all'] {
 	display: block;
 }
 
-#todo-list li .edit {
-	display: none;
-}
+/*#todo-list li .edit {*/
+	/*display: none;*/
+/*}*/
```

Comment:
Ok. I'm not really clear on how your repo works, I was under the impression that each of the frameworks in the examples folder stood completely on its own, but that doesn't sound like the case.

I suppose I could leave the CSS alone and let it do it the non-Meteor way. As long as the folder/file structure is left alone though. That would probably be the best route. Really, this code needed a serious overhaul. It was a mess, outdated, and not very Meteor-like.